### DISCUSSION TITLE
Sample vector.toml file

### DISCUSSION THREAD

**Discussion by emlim23** [vectordotdev/vector#17234] [Q&A]
Hi all, 

I need help with my use case: We have a 3rd-party server/service that uses Vector to enable exporting of logs. Are currently using Datadog already and we would like to connect our datadog instance with their exporter using vector.toml file. Can somebody provide some examples on how we can do this?

We're using CubeJS and in their docs they have this: https://cube.dev/docs/cloud/workspace/logs 
I'm not familiar with Vector so I'm not sure what I can put in the vector.toml file they provided to enable it to connect to our Datadog instance. Thanks!

**Comment by jszwedko** [comment]
Hi @emlim23 ,

I imagine the 3rd-party can assist you better since you are unfamiliar with Vector, but if you want to try it yourself, I'd recommend looking at Datadog sinks you see on https://vector.dev/docs/reference/configuration/sinks/ to see how to configure them.
